mailman/submit.py

```diff
@@ -61,15 +62,18 @@ def set_status(status):
         sub.save()
         return 0
 
+    deadline_buffer = timedelta(minutes=7)
```

Comment:
this should be configurable rather than hardcoded, either as an env variable or config file